### PR TITLE
Limit the maximum size of the training and test sets

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Signed-off-by: Vadim Markovtsev <vadim@sourced.tech>

### CODE REVIEW

File: lookout/style/format/analyzer.py [hunk 1]
```diff
@@ -101,6 +101,7 @@ class FormatAnalyzer(Analyzer):
             "test_dataset_ratio": 0.0,
             "line_length_limit": 500,
             "lower_bound_instances": 500,
+            "overall_size_limit": 2 << 20,  # 2 MB
```
Comment (zurk) [inline lookout/style/format/analyzer.py]: why do you set exactly this number?

Comment (vmarkovtsev) [reply lookout/style/format/analyzer.py]: Taken from jQuery. I guess everything which is bigger starts to cause trouble.

**Comment by vmarkovtsev** [comment]
@m09 Can you please relaunch the min_samples_leaf experiments with this PR merged - it is going to change the numbers.

**Comment by m09** [comment]
sure!

**Pull request merged**